helm/botkube/values.yaml

```diff
@@ -69,31 +69,32 @@ sources:
       resources:
```

Comment:
BTW PR title says `rename include to exclude` 😄 but I think you meant `rename ignore to exclude` 😄